README.md

```diff
@@ -19,36 +19,39 @@ this model.
 
 ## Preparing the software environment
 
-You need to install a number of packages to run the model. You can use the
-following julia code to do so:
+You first need to connect your julia installation with the central
+[Mimi registry](https://github.com/mimiframework/MimiRegistry) of Mimi models.
+This central registry is like a catalogue of models that use Mimi that is
+maintained by the Mimi project. To add this registry, run the following
+command at the julia package REPL:
 
-````julia
-using Pkg
-Pkg.add("Mimi")
-Pkg.add("ExcelReaders")
-````
+```julia
+pkg> registry add https://github.com/mimiframework/MimiRegistry.git
+```
+
+You only need to run this command once on a computer.
 
-You will also need to add the Mimi registry and this package.
+The next step is to install MimiRICE2010.jl itself. You need to run the
+following command at the julia package REPL:
+
+```julia
+pkg> add MimiRICE2010
 ```
-registry add https://github.com/mimiframework/MimiRegistry.git
-Pkg.add("MimiRICE2010")
+
+You probably also want to install the Mimi package into your julia environment,
+so that you can use some of the tools in there:
+
+```julia
+pkg> add Mimi
 ```
 
 ## Running the model
 
 The model uses the [Mimi framework](https://github.com/mimiframework/Mimi.jl)
 and it is highly recommended to read the Mimi documentation first to
 understand the code structure. For starter code on running the model just once,
-see the code in the file ``examples/main.jl``.
+see the code in the file `examples/main.jl`.
 
-The current exported functions include:
-```
-constructrice
-getrice
-getrice2010parameters
-getmarginal_rice_models
-getmarginaldamages_rice
-```
```

Comment:
@lrennels I removed these for now. I don't want to set expectations that we will keep this API, so maybe lets just not talk about it for now.

Reply:
Ok sounds good, are they used anywhere that we need to add prefixes to now?

Reply:
I don't think so... Probably by @FrankErrickson. I think it is still ok to export them, I just don't want to advertise that particular API too much, given that we will break it once we have a unified story for this stuff across all models.

Reply:
Ah I see ok

Reply:
Ok all looks good!